lib/techniqueAttributeForParameter.js

```diff
@@ -1,6 +1,13 @@
 'use strict';
 module.exports = techniqueAttributeForParameter;
 
+/**
+ * Retrieves the primitive attributes that has a given parameter name.
+ *
+ * @param {Object} technique A javascript object containing a glTF technique.
+ * @param {String} parameterName The search string for parameter.
+ * @returns {Array.<string>} The primitive attribute with matching paramerterName.
```

Comment:
This function returns a single string. Change the description and return type.

"paramerterName" -> "parameterName"